tidoop-mr-lib-api/src/server.js

```diff
@@ -23,36 +23,134 @@
  * Author: frb
  */
 
-// requires
+// module dependencies
 var Hapi = require('hapi');
-var config = require('../conf/tidoop-mr-lib-api.json');
-var pjson = require('../package.json');
+var run = require('./cmd_runner.js').run;
+var version = require('../package.json').version;
+var port = require('../conf/tidoop-mr-lib-api.json').port;
+var tidoopMRLibPath = require('../conf/tidoop-mr-lib-api.json').tidoopMRLibPath;
```

Comment:
Instead of requiring the same file over and over again, you should require it once and reuse that object. require() loads (it uses a cache) and compiles the file each time it is invoked... ;)